src/main/drivers/dshot.c

```diff
@@ -45,6 +45,10 @@
 
 #include "fc/rc_controls.h" // for flight3DConfig_t
 
+#include "flight/mixer.h"
+
+#include "sensors/esc_sensor.h"
```

Comment:
This include is no longer needed

Reply:
Removed, also in the other files that no longer need it